packages/lb-annotation/src/core/scheduler.ts

```diff
@@ -312,6 +312,21 @@ export class ToolScheduler implements IToolSchedulerOperation {
     return this.toolOperationList[lastOneIndex];
   }
 
+  public updateDataByToolName(toolName: EToolName, result: any) {
```

Comment:
Add more moment to explain something like this function is not support to the same tool?

Reply:
added